src/components/ClaimForm.js

```diff
@@ -179,7 +184,7 @@ class ClaimForm extends Component {
       if (!!this.state.claim.items) {
         items = [...this.state.claim.items];
         if (!this.props.forReview) items.pop();
-        if (items.length && items.filter((i) => !this.canSaveDetail(i, "item")).length) {
+        if (items.length && items.filter((i) => !this.canSaveDetail(i, "item", forReview=forReview)).length) {
```

Comment:
It should be enough to pass forReview here, no need to pass forReview=forReview.